cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -15,44 +15,33 @@ This Cheat Sheet provides guidance on the various areas that need to be consider
 
 ### Hashing vs Encryption
 
-Hashing and encryption are two terms that are often confused or used incorrectly. The key difference between them is that hashing is a **one way** function (i.e, it is not possible to "decrypt" a hash and obtain the original value), whereas encryption is a two-way function.
+Hashing and encryption both provide ways to keep sensitive data safe. However, in almost all circumstances, **passwords should be hashed NOT encrypted.**
 
-In almost all circumstances, passwords should be hashed rather than encrypted, as this makes it difficult or impossible for an attacker to obtain the original passwords from the hashes.
+**Hashing is a one-way function** (i.e, it is not possible to "decrypt" a hash and obtain the original plaintext value). Hashing is appropriate for password storage because even if an attacker obtains the hashed password, they can't enter it into the password field of your application and log in as the victim.
 
-Encryption should only be used in edge cases where it is necessary to be able to obtain the original password. Some examples of where this might be necessary are:
+**Encryption is a two-way function**, meaning that the original plaintext can be retrieved. Encryption is appropriate for storing data such as a user's address since this data is displayed in plaintext on the user's profile. Hashing their address would result in a garbled mess.
 
-- If the application needs to use the password to authenticate against an external legacy system that doesn't support SSO.
-- If it is necessary to retrieve individual characters from the password.
```

Comment:
I could see this getting added back in if someone is able to provide an example of why this might be necessary. I couldn't think of one.